### DISCUSSION TITLE
WordPress homepage dynamic display error

### DISCUSSION THREAD

**Discussion by spencersmb** [WebDevStudios/nextjs-wordpress-starter#732] [Q&A]
I've followed the steps and seem to be getting a build done, but when I load the local host preview using "npm run dev" - the homepage displays this:

_To display your WordPress homepage dynamically, set your homepage to a static page via the WP dashboard (Settings: Reading Settings)._

I've tripple checked the LOCAL site and it definitely has the STATIC PAGES set to HOMEPAGE and BLOG so I'm not 100% sure how to fix this. Any ideas I would sure appreciate it!

**Comment by spencersmb** [comment]
So I think I fixed it but I have a few questions. In my .env file I've set my Wordpress URL to the URL from my LOCAL test site. I'm assuming that I will env variables through next/vercel to define the production URL to connect the WPENGINE site?

Thanks

**Reply by tomthor** [reply]
Hi, i'm also having this problem now, any ideas how to fix it?
